### PR TITLE
Move back docker event fields to it's namespace

### PR DESCRIPTION
**Pull request by exekias**
Follow-up of https://github.com/elastic/beats/pull/9856. We realized changing the event format from the original one may mislead users

### CODE REVIEW

File: metricbeat/module/docker/event/event.go [hunk 1]
```diff
@@ -122,15 +122,11 @@ func (m *MetricSet) reportEvent(reporter mb.PushReporterV2, event events.Message
...
-				"action":  event.Action,
-				"created": time,
-				"id":      event.ID,
-			},
-		},
 		MetricSetFields: common.MapStr{
+			"id":     event.ID,
+			"type":   event.Type,
+			"action": event.Action,
+			"time":   event.Time,
```
Comment (ruflin) [inline metricbeat/module/docker/event/event.go]: `time` is probably the one I would move to ECS. I wonder if this should even be `@timestamp`?

Comment (exekias) [reply metricbeat/module/docker/event/event.go]: yes, I will just remove it from here, as it's already reported under @timestamp here: https://github.com/elastic/beats/blob/48d2295fdaea431e59b2d6b42a59044de02d5722/metricbeat/module/docker/event/event.go#L124?

**Comment by exekias** [comment]
ouch, done!

File: metricbeat/module/docker/event/_meta/data.json [hunk 2]
```diff
@@ -6,23 +6,21 @@
...
             "actor": {
                 "attributes": {
-                    "image": "busybox",
-                    "name": "wizardly_knuth"
+                    "name": "busybox"
                 },
-                "id": "9d4c3af1d4a0268940205aad04a3c6bf434b32f45e0784dec5c283201ef299fc"
+                "id": "busybox:latest"
             },
-            "from": "busybox",
```
[the remark this answers is not in the document]
Comment (exekias) [reply metricbeat/module/docker/event/_meta/data.json]: yes, it takes many forms, you also get events when creating a network, so this field is not always a image

**Pull request merged**